velox/exec/tests/utils/PlanBuilder.cpp

```diff
@@ -170,13 +172,25 @@ core::PlanNodePtr PlanBuilder::TableScanBuilder::build(core::PlanNodeId id) {
     }
 
     if (!hasAssignments) {
+      auto columnType = HiveColumnHandle::ColumnType::kRegular;
+      if (partitionKeys_.count(name) > 0) {
+        columnType = HiveColumnHandle::ColumnType::kPartitionKey;
+      }
+      HiveColumnHandle::ColumnParseParameters columnParseParameters;
+      if (tableFormat_ == "iceberg" && type->isDate()) {
```

Comment:
Let's not add format-specific features here. Let's create the assignments and add them to the test.